cscs-checks/prgenv/gpu_direct_acc.py

```diff
@@ -18,10 +18,11 @@ def __init__(self, **kwargs):
             self.num_tasks_per_node = 1
         elif self.current_system.name in ['kesch']:
             self.modules = ['craype-accel-nvidia35']
-            self._pgi_flags = '-acc -ta=tesla:cc35 -Mnorpath'
+            self._pgi_flags = '-acc -ta=tesla:cc35'
             self.variables = {
                 'MPICH_RDMA_ENABLED_CUDA': '1',
                 'MV2_USE_CUDA': '1',
+                'G2G': '1'
                 'G2G': '1',
```

Comment:
@teojgo This does not fix the conflict; it breaks the code... I am fixing it now. You should not rely entirely on the CI to catch errors in user tests, especially after merges.